### PR TITLE
First Part

### PR DESCRIPTION
**Pull request by ChuckConde**
There is a comment in the 6.html

### CODE REVIEW

File: 00-HTML-CSS-basics/6.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <link rel="stylesheet" type="text/css" href="6.css">
+</head>
+
+<body>
+    <h1>Sign Up For Free!</h1>
+    <form id="signup-form">
+
+        <div>
+
+            <input type="text" name="name" id="name" value="" placeholder="First Name" required>
```
Comment (pabloascencao) [inline 00-HTML-CSS-basics/6.html]: `value="" ` esto no es requerido en los inputs a menos que quieras pasar algun valor por defecto al input

File: 00-HTML-CSS-basics/6.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                <option value="jockey">Jockey</option>
+            </select>
+        </div>
+        <div>
+
+            <textarea name="comments" id="comments" cols="40" rows="10" placeholder="Bio" required></textarea>
+        </div>
+        <div>
+            <input type="submit" value="submit" />
+            <input type="reset" value="reset" />
```
Comment (pabloascencao) [inline 00-HTML-CSS-basics/6.html]: Para funcionalidades de boton es mejor usar los tag `<button></button>`

File: 00-HTML-CSS-basics/5.html [hunk 3]
```diff
@@ -8,7 +8,7 @@
 </head>
 <body>
     <div>
-    <img src="Wolf.jpg" alt="Wolf">
+    <img src="Wolf.jpg" alt="A Wolf in Winter">
```
Comment (pabloascencao) [inline 00-HTML-CSS-basics/5.html]: winter is comming?

Comment (pabloascencao) [inline 00-HTML-CSS-basics/5.html]: winter is comming?

**Comment by ChuckConde** [comment]
Non seeing people gotta be aware of their allies against the white walkers

**Pull request merged**